src/components/Show.tsx

```diff
@@ -0,0 +1,46 @@
+import Link from "next/link";
+import ShowDetail from "./ShowDetail";
+
+type Row = {
```

Comment:
Row と RowType があってまぎらわしいのでこっちを ShowProps とかにしてみるのはいかがでしょうか？

```suggestion
type ShowProps = {
```

Reply:
https://github.com/tatata05/next-app/pull/7/commits/35276864bab4938543690c340091d6251b909ed6
修正しました！

Reply:
あ、逆です RowType はそのままで良いです。意味的にこっちは Row じゃないと思うので。